src/index.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-alert, import/no-extraneous-dependencies */
+import 'bootstrap';
+import './styles/style.scss';
+import { getTemp } from './scripts/getTemp';
+import domPopulator from './scripts/domPopulator';
+
+getTemp().then(domPopulator);
+
+const searchBtn = document.querySelector('form');
+searchBtn.addEventListener('submit', (e) => {
+  const searchValue = document.querySelector('#searchInput').value;
+  const metricValue = document.querySelector('#inputMetric').value;
+  e.preventDefault();
+  getTemp(searchValue, metricValue)
+    .then((res) => {
+      domPopulator(res, metricValue);
+    })
+    .catch(() => {
+      alert('location you entered is not valid.');
```

Comment:
- [ ] You should never use alerts in modern Javascript web applications. A lot of browsers block them and they are nearly impossible to style or to customize. 
[Check this thread about it for more.](https://stackoverflow.com/questions/20359807/why-shouldnt-i-use-alert-in-javascript)
Please show the message in a component at the DOM instead. You can build something that only shows up when there's this error and change its classes/styles and innerText so it looks right.